src/Components/AddVideoToPlayList/AddNewPlayListButton.js

```diff
@@ -0,0 +1,29 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../context/auth-context";
+import { usePlaylistModal } from "../../context/playlist-context";
+
+const AddNewPlayListButton = () => {
+  const { modalDispatch } = usePlaylistModal();
+  const {
+    authState: { login },
+  } = useAuth();
+  const navigate = useNavigate();
+  return (
+    <div
+      className='playlist'
+      onClick={() =>
+        login
+          ? modalDispatch({
+              type: "SHOW_HIDE_NEW_PLAYLIST_INPUT_FIELD",
+              value: true,
```

Comment:
it is better to put values inside payload object in dispatch function. LIke this -
dispatch({type: "SOME_TYPE",payload:{value:true}})